docs/examples/use_cases/tensorflow/yolov4/Dockerfile

```
@@ -0,0 +1,12 @@
+FROM nvcr.io/nvidia/tensorflow:21.06-tf2-py3
+WORKDIR yolov4
+
+RUN apt clean autoclean
+RUN python -m pip install --upgrade pip
+RUN apt update
+RUN apt install python3-opencv  -y
+
+COPY requirements.txt requirements.txt
+RUN pip3 install -r requirements.txt
```

Please don't create more docker layers that it is needed
```suggestion
RUN apt clean autoclean && \
         apt update && apt install -y python3-opencv && \
         rm -rf /var/lib/apt/lists/*

COPY requirements.txt requirements.txt
RUN python -m pip install --upgrade pip && \
         pip3 install -r requirements.txt && \
         rm -rf /root/.cache/pip/
```